app/services/clipboard.ts

```diff
@@ -4,63 +4,69 @@ import { SourcesService } from './sources';
 import { shortcut } from './shortcuts';
 import { Inject } from '../util/injector';
 import { SourceFiltersService } from './source-filters';
+import { SelectionService } from 'services/selection';
 
 
 interface IClipboardState {
-  sourcesIds: string[];
+  sceneItemsIds: string[];
   filtersIds: string[];
```

Comment:
I know you didn't change this, but correct grammar would be `filterIds` to be consistent.